pom.xml

```diff
@@ -918,6 +918,11 @@
           <artifactId>maven-bundle-plugin</artifactId>
           <version>5.1.1</version>
         </plugin>
+        <plugin>
+          <groupId>org.cyclonedx</groupId>
+          <artifactId>cyclonedx-maven-plugin</artifactId>
+          <version>2.7.3</version>
```

Comment:
```suggestion
          <version>2.7.4</version>
```

Reply:
The cyclonedx-maven-plugin-2.7.4 was released last week, more information - https://github.com/CycloneDX/cyclonedx-maven-plugin/releases/tag/cyclonedx-maven-plugin-2.7.4

Reply:
Actually, I found it doesn't work in some cases, @VinodAnandan . While helping other projects, I tried and reverted to back 2.7.3.
- https://cwiki.apache.org/confluence/display/COMDEV/SBOM

Could you try that in a separate PR after this PR?

Reply:
@dongjoon-hyun sure, I will try 2.7.4 update in a separate PR after this PR.

Reply:
Thank you so much, @VinodAnandan !

Reply:
@dongjoon-hyun 

> Actually, I found it doesn't work in some cases

I'm working on improving the plugin, can you point me where 2.7.3 works but not 2.7.4? I was not supposed to introduce any regression :)

@VinodAnandan if you worked on this, I'm interested to help